app.js

```diff
@@ -1,14 +1,19 @@
 'use strict';
 
 var hook = require('./hook');
+var config = require('./config');
+var logger = require('./logger');
+var path = require('path');
```

Comment:
Just a small thing. In other files, we've put contributed modules above custom modules separated by one line.

https://github.com/mdn/discord/blob/master/processor.js#L3-10